src/commands/generateRunbook.ts

```diff
@@ -89,13 +98,15 @@ export const generateRunbook = (secrets: vscode.SecretStorage) => vscode.window.
         await verifyHasOpenAIKey(secrets, true);
     }
 
-    const promptTypes = getPromptTypes();
+    const promptOption = await getPromptForPrompt();
 
-    const promptOption = await vscode.window.showQuickPick(promptTypes.map(f => ({ label: f.title, detail: f.title, index: f.type })), { title: "Select prompt type", ignoreFocusOut: true });
+    if (!promptOption) {
+        return;
```

Comment:
I didn't understand the name getPromptForPrompt at first. But this combines the entire flow of a user selecting and generating prompts so this must involve some generic docker engine failure handling too, right?

Reply:
This just handles the quickpick, I can change the name